breathe/renderer/sphinxrenderer.py

```diff
@@ -950,26 +950,32 @@ def visit_typedef(self, node):
         return self.render_declaration(node, declaration, objtype=obj_type,
                                        update_signature=self.update_signature)
 
+    def update_signature_with_initializer(self, signature, node):
+        initializer = node.initializer
+        if initializer:
+            nodes = self.render(initializer)
+            separator = ' '
+            if not nodes[0].startswith('='):
+                separator += '= '
```

Comment:
This is not always desirable, e.g. when the variable is brace-initialized:
```cpp
constexpr const char* description{R"descriptive text"};
```
-->
```xml
...
        <type>constexpr const char *</type>
        <definition>constexpr const char* description</definition>
        <argsstring></argsstring>
        <name>description</name>
        <initializer>{ R&quot;descriptive text&quot; }</initializer>
```
I don't actually have access to the rst, or intermediate docutils ast, but I guess the output in rst is something like:
```rst
.. code-block::
   constexpr const char* description = {R"descriptive text"}
```
There are two issues with this:
1. additional `=`
2. missing `;`

This fails like:
```shell
filename.rst:4:Invalid C++ declaration: Error in braced-init-list, expected ',' or '}'. [error at 47]
  static constexpr const char * description   = { R"descriptive text" }
  -----------------------------------------------^
```